index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>portfolio</title>
+</head>
+<body>
+  <header class="navigation">
+    <li class="logo">My Logo</li>
```

Comment:
- [ ] Kindly use an `a` tag instead of `li` tag for this element as suggested by the previous reviewers. The `a` tag improves user experience and it is considered a good practice to put logos in `a` tag. :+1: 

You can implement it as:
```suggestion
<a href="#" class="logo">My Logo</a>
```